{
    "id": 237297,
    "name": "CVE-2024-1141",
    "cwe": "NVD-CWE-noinfo",
    "modified": "2024-05-22T20:15:08",
    "published": "2024-02-01T14:15:08",
    "status": "Modified",
    "summary": "A vulnerability was found in python-glance-store. The issue occurs when the package logs the access_key for the glance-store when the DEBUG log level is enabled.",
    "cvss3": {
        "cvss": 5.5,
        "exploitability_score": 1.8,
        "impact_score": 3.6,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
    },
    "cvss2": null,
    "cpes": []
}